core/src/main/java/uk/gov/onelogin/core/tokens/domain/save/SaveToEncryptedSecureStore.kt

```diff
@@ -0,0 +1,14 @@
+package uk.gov.onelogin.core.tokens.domain.save
+
+fun interface SaveToEncryptedSecureStore {
```

Comment:
Maybe a better name here would be 
```suggestion
fun interface SaveToTokenSecureStore {
```